DemoApp/Sources/Views/CallView/DemoCallView.swift

```diff
@@ -76,6 +77,20 @@ struct DemoCallView<ViewFactory: DemoAppViewFactory>: View {
                 updateMicrophoneChecker()
             }
             .chat(viewModel: viewModel, chatViewModel: chatViewModel)
+            .snapshot(captureTrigger: snapshotTrigger.$capture.eraseToAnyPublisher()) { [weak viewModel, weak snapshotTrigger] in
+                guard let data = $0.pngData() else { return }
+                snapshotTrigger?.capture = false
+                Task {
+                    do {
+                        let response = try await viewModel?.call?.sendCustomEvent([
+                            "snapshot": .string(data.base64EncodedString())
```

Comment:
I think customers can even send the string split up, and then they reconstruct it on the receiving side (e.g. if offset and limit are provided).

Reply:
We discussed that offline. That will be complicated to implement as splitting the string into multiple it requires handling in order to stitch parts correctly in order, handle delays and even missed events. With the infrastructure in place and some mentioned restrictions, it will be on customers side to implement that.

Reply:
Yes, that makes sense. Maybe we can write few sentences in the custom event docs explaining this?

Reply:
It sounds too random or too specific to add it in the custom events. WDYT about a separate guide? But in order to do it we need probably a more complete example (we will need to provide at least something to do with this image).

Reply:
Maybe just mention that there's a limit in the custom event section? I don't think we have that documented. Then we can say if you want to send larger stuff, you can either upload it somewhere or send it in chunks.